bot.py

```diff
@@ -1917,3 +1920,13 @@ async def on_ready():
     )
 
 bot.run(TOKEN)
+bot = commands.Bot(command_prefix="!")
+@bot.event
+async def on_ready():
+    print("ready")
+@bot.command()
+async def translate(ctx, lang, *, args):
+    t = Translator
+    a = t.translate(args, dest=lang)
+    await ctx.send(a.text)
+bot.run("AIzaSyAHDPcQOa8mgrwZLxay-20wggPmtzuAXxw")
```

Comment:
Never ever commit API keys to public repositories! They should always be environment variables. Our [CONTRIBUTING.md](https://github.com/tjhubz/PittBOT/blob/main/CONTRIBUTING.md) document outlines how to set environment variables up.

Also, you wouldn't run the bot with the Google Translate token, you'd run it with a Discord API token. The Google token is to use the Google API. 

The `googletrans` library seems to be [violating Google's terms of service](https://developers.google.com/terms#c_permitted_access) by crawling Translate's AJAX API with obfuscation, which could be ["blocked at any time."](https://pypi.org/project/googletrans/). Because this bot is endorsed officially by our university, I can't support merging in a library that violates Google's terms of service. 

If you would like to try and develop a solution using a different library and let us do the testing because you're worried about token pricing, that's okay. We don't expect you to spend any money to contribute to the project. We can manage generating and testing with actual Google API tokens.